packages/app/src/settings/project/Experiments.vue

```diff
@@ -54,13 +54,33 @@ const props = defineProps<{
 }>()
 
 const localExperiments = computed(() => {
-  return props.gql?.config ? (props.gql.config as CypressResolvedConfig).filter((item) => item.field.startsWith('experimental')).map((configItem) => {
+  // get experiments out of the config
+  const experimentalConfigurations = props.gql?.config ? (props.gql.config as CypressResolvedConfig).filter((item) => item.field.startsWith('experimental')) : []
+
+  // get experimental retry properties on the 'retries' config object. Mutate the experimentalConfigurations array as to not have side effects with props.gql.config
+  // TODO: remove this once experimentalRetries becomes GA. This is to be treated as a one off as supported nested experiments inside config is rare.
+  const { value: { experimentalStrategy, experimentalOptions, from } } = props.gql?.config.find((item) => item.field === 'retries')
```

Comment:
Should we match line 58 just to provide some type safety on the find callback?

```suggestion
  const { value: { experimentalStrategy, experimentalOptions, from } } = (props.gql?.config as CypressResolvedConfig).find((item) => item.field === 'retries')
```